packages/plugin-types/src/aws_client_provider.ts

```diff
@@ -0,0 +1,9 @@
+import { Client, MetadataBearer } from '@smithy/types';
```

Comment:
sorry for the churn on this one, it looks like these types are re-exported by @aws-sdk/types which is what we should use here